### DISCUSSION TITLE
已经装了Numpy，但是仍然报错 ERROR 'numpy.core.multiarray failed to import'

### DISCUSSION THREAD

**Discussion by Logan66666** [magicpdf/Magic-PDF#642] [Q&A] [answered]
跟着教程走到“第一次运行前测试”

命令行中用了 magic-pdf -p small_ocr.pdf -o "C:\Users\LOG" -m auto
![image](https://github.com/user-attachments/assets/8b4ab156-a71e-4768-978f-adc2df68cdfe)

出现下述报错
![1726933171252](https://github.com/user-attachments/assets/3dd2ea8d-7920-49eb-8453-11417b985dcc)

numpy的版本是2.0.1
尝试删了再装也没用

求大佬解答~

**Comment by myhloli** [comment, marked as answer]
numpy2.0应该是不兼容的，这一点在依赖列表中已经指定了numpy的版本，不会安装到2.0.1

**Reply by Logan66666** [reply]
> numpy2.0应该是不兼容的，这一点在依赖列表中已经指定了numpy的版本，不会安装到2.0.1

我用了这个：
pip install magic-pdf[full] --extra-index-url https://myhloli.github.io/wheels/

运行结果中看这行应该是≥1.21.6就算满足了，所以高版本不兼容的情况就漏过去了。
![image](https://github.com/user-attachments/assets/e4958c34-caf1-4317-8ddb-7a58bd04b7d6)

我退回到1.26.4确实这里就正常了，感谢大佬！（不过出现新问题了明天继续研究）

**Comment by Logan66666** [comment]
您好，邮件已收到。谢谢！
